calculator/operations.py

```diff
@@ -1,18 +1,17 @@
+from functools import reduce
 
 def add(*args):
-    # your implementation here
-    pass
-
+    return reduce((lambda x, y: x + y), args)
```

Comment:
Aha! So that is how you do the calculations with a lambda. I just couldn't figure it out :(